governance/pyth_staking_sdk/src/pyth-staking-client.ts

```diff
@@ -372,6 +372,45 @@ export class PythStakingClient {
     return sendTransaction(instructions, this.connection, this.wallet);
   }
 
+  public async unstakeFromAllPublishers(
```

Comment:
lgtm. The only concern is that there might be too many transactions causing the block hash to expire. We have similar issue in claimRewards. But I think the fix should be in solana utils. cc: @guibescos

Reply:
thats fair but this should be a pretty rare edge case anyways since it only shows up in the UI if the user is in restricted mode (i.e. in a geofenced country) AND has ois positions (i.e. circumvented geoblocking somehow) so I'm not too worried.  But if there's a better way to do it let me know.

Reply:
These users are not likely to have that many positions tbh.